### ISSUE TITLE
Traefik 2.0 logging is messy

### ISSUE THREAD

**Issue by trondhindenes** [containous/traefik#5750]
Because of some misconfigurations, we're seeing the following in Traefik logs:
```
time="2019-10-29T09:43:01Z" level=error msg="Cannot create service: subset not found" namespace=dev-sre-default serviceName=common-logging-kibana-authproxy-v1-dev-main servicePort=8080 providerName=kubernetescrd ingress=common-logging-kibana-authproxy-v1-dev-main-httpredirect-0
time="2019-10-29T09:43:01Z" level=error msg="Cannot create service: subset not found" namespace=dev-sre-default ingress=common-logging-kibana-authproxy-v1-dev-main-https-0 serviceName=common-logging-kibana-authproxy-v1-dev-main servicePort=4180 providerName=kubernetescrd
time="2019-10-29T09:43:01Z" level=error msg="Cannot create service: subset not found" ingress=common-logging-kibana-authproxy-v1-dev-main-httpredirect-0 servicePort=8080 namespace=dev-sre-default providerName=kubernetescrd serviceName=common-logging-kibana-authproxy-v1-dev-main
time="2019-10-29T09:43:01Z" level=error msg="Cannot create service: subset not found" servicePort=4180 namespace=dev-sre-default serviceName=common-logging-kibana-authproxy-v1-dev-main providerName=kubernetescrd ingress=common-logging-kibana-authproxy-v1-dev-main-https-0
time="2019-10-29T09:43:01Z" level=error msg="Cannot create service: subset not found" providerName=kubernetescrd namespace=dev-sre-default servicePort=4180 ingress=common-logging-kibana-authproxy-v1-dev-main-https-0 serviceName=common-logging-kibana-authproxy-v1-dev-main
time="2019-10-29T09:43:01Z" level=error msg="Cannot create service: subset not found" namespace=dev-sre-default serviceName=common-logging-kibana-authproxy-v1-dev-main servicePort=8080 providerName=kubernetescrd ingress=common-logging-kibana-authproxy-v1-dev-main-httpredirect-0

```

Notice that the attributes are in random order. This makes it very hard to parse the log files. It would be much better if attributes were sorted by attribute name so that the order of fields were consistent - it would make the logs much easier to read.

**Comment by traefiker** [comment]
Hi! I'm Træfiker :robot: the bot in charge of communication regulation.

Thanks for your interest in Traefik!

Issue templates help us help you by providing all necessary information.

Please edit your issue and use the available templates:

- [Feature request](https://raw.githubusercontent.com/containous/traefik/master/.github/ISSUE_TEMPLATE/Feature_request.md)
- [Bug report](https://raw.githubusercontent.com/containous/traefik/master/.github/ISSUE_TEMPLATE/Bug_report.md)

And remember: each time someone ignores the template, a cute little bunny dies.
<details>
<summary>#SaveTheCuteBunny :heart: :rabbit: :heart:</summary>

![southpark - estrella kill rabbit](https://user-images.githubusercontent.com/5674651/27143914-3cea7896-5130-11e7-81f8-02d8ac2da7a7.gif)

![southpark - estrella kill rabbit2](https://user-images.githubusercontent.com/5674651/27143924-49104c72-5130-11e7-82ae-ccedbd5dfa1c.gif)

</details>

**Comment by traefiker** [comment]
Hi! I'm Træfiker :robot: the bot in charge of tidying up the issues.

I have to close this one because of its lack of activity :disappointed:

Feel free to re-open it or join our [Community Forum](https://community.containo.us/).
